examples/key_release/payload.py

```diff
@@ -2,9 +2,29 @@
 # Licensed under the MIT License.
 
 from http.server import BaseHTTPRequestHandler, HTTPServer
+import os
+import subprocess
 
 import requests
 
+def grpc_ready_test():
```

Comment:
I wonder if having two distinct example, one using the REST endpoint, one using the GRPC one, would be easier for a user to understand.

Reply:
I believe that's what this [set of tests](https://github.com/microsoft/confidential-aci-examples/blob/87659ba3793904d897b734a4a78f1de5091a8067/examples/key_release/test.py#L20) is doing